snuba/web/query.py

```diff
@@ -165,26 +166,33 @@ def _run_and_apply_column_names(
         reader,
     )
 
-    name_alias_mappings = [
-        (select.name, select.expression.alias)
-        for select in clickhouse_query.get_selected_columns_from_ast()
-    ]
-    discrepancies = [
-        (name, alias) for name, alias in name_alias_mappings if name != alias
-    ]
-    if discrepancies:
-        logger.warning(
-            "Discrepancies between select clause names and aliases",
-            extra={"mappings": discrepancies},
-            exc_info=True,
-        )
```

Comment:
This never logged a single message (in weeks)  so we are safe that there are no mismatches between names and aliases and, as a consequence, that there is a strict 1:1 relationship between name and alias.